arch/X86/X86Mapping.c

```diff
@@ -4040,13 +4040,13 @@ static insn_map insns[] = {	// full x86 instructions
 	{
 		X86_CLC, X86_INS_CLC,
 #ifndef CAPSTONE_DIET
-		{ 0 }, { 0 }, { 0 }, 0, 0
+		{ 0 }, { X86_REG_EFLAGS }, { 0 }, 0, 0
```

Comment:
https://github.com/aquynh/capstone/pull/798 ; 

X86_REG_EFLAGS was added to X86_STC and X86_STD in the table of full x86 instructions, and for the table of reduced one (if macro CAPSTONE_X86_REDUCE is defined or not).

Sorry for the trailing zero, will fix